Assignment 6

```diff
@@ -59,10 +65,11 @@ public class CollatzCalculatorServer {
         }
     }
 
-    public static void main(String[] args) throws Exception {
+    public static void main(String[] args) throws Exception 
+    {
         int port = 50051;
-        CollatzCalculatorServer server = new CollatzCalculatorServer(port);
+        CollatzCalServer server = new CollatzCalServer(port);
         server.start();
-        server.server.awaitTermination();
+        server.server.awaitTerm();
```

Comment:
I don't think this new version will compile - the method on the grpc Server class is called awaitTermination